packages/next/src/build/webpack/loaders/next-app-loader/index.ts

```diff
@@ -50,6 +50,7 @@ export type AppLoaderOptions = {
   nextConfigOutput?: NextConfig['output']
   nextConfigExperimentalUseEarlyImport?: true
   middlewareConfig: string
+  isGlobalNotFoundEnabled: true | undefined
```

Comment:
why not just keep usual optional boolean here?

```suggestion
  isGlobalNotFoundEnabled?: boolean
```

Reply:
webpack loader will serialize the boolean `false` to string `"false"`, I also don't want to use optional to make this easily being forgotten. Hence choose the `true | undefined` here.